### ISSUE TITLE
2022.05 FPP Connect hangs for a while before upload

### ISSUE THREAD

**Issue by sslupsky** [smeighan/xLights#3143]
**Describe the bug**
Noticed a weird thing today with 2022.05 where when using FPP Connect, there is a rather long delay between when you click upload and the dialog showing the upload progress appears.  During this time, the cursor is a beachball.  I checked my controllers and they are all online.

**To Reproduce**
(and/or link to a short video showing the problem with audio track describing what you are doing)
1. Tools->FPP Connect
2. Select a sequence (I selected three short 90 second animated sequences)
3. Click upload

**Comment by bcarriveau** [comment]
Cannot recreate ,on xLights master of today and pushing to FPP 6.x-Master. new 90sec animation and existing sequences. Just the very long discovery with offline controllers.

**Comment by sslupsky** [comment]
I will try to put more info together on this.  I am able to recreate at my end.  I checked my controllers and they were all online.

**Comment by sslupsky** [comment]
Note, my FPP version is 5.5.  I upgraded recently from 5.4 so perhaps that is the issue.

**Comment by derwin12** [comment]
Do you have a lot of playlists? 
Do you have a usb stick on the computer that has xlights on it (it will think that is a controller).
Check tools->View Logs see if there are any clues there.

**Comment by sslupsky** [comment]
I will have a look at the logs.  I used the previous version of xLights on another computer (2022.04-1) and that worked fine so I think it likely it is not due to an issue with FPP 5.5.

**Comment by sslupsky** [comment]
No USB sticks in use.  I have eight playlists on the master.  For some reason I am getting an error when trying to view the xLights log file:
![Screen Shot 2022-02-28 at 10 02 31 AM](https://user-images.githubusercontent.com/16888734/156025979-4fa6c97f-bc8b-40b6-a2b3-92634b01427d.png)


I spent some additional time on this and identified that the problem appears to be related to a single remote.  I have three controllers, all three are online.  One master and two remotes.  If I try FPP connect with each controller separately, I have determined that the issue appears to manifest with one specific remote.  It is not clear why that remote would work fine with xLights on one computer but act weird with a different computer.  No errors or otherwise anything suspicious in the fppd log or syslog on the remote.

I will close for now since this doesn't appear to be an xLights issue that I can identify.  I'll open again if I figure out what is causing this and I determine that it is related to xLights.
